easybuild/easyconfigs/h/HAL/HAL-2.1-foss-2020b.eb

```diff
@@ -0,0 +1,116 @@
+easyblock = 'MakeCp'
+
+name = 'HAL'
+version = '2.1'
+
+homepage = 'https://github.com/ComparativeGenomicsToolkit/hal'
+description = """HAL is a structure to efficiently store and index multiple genome alignments
+and ancestral reconstructions. HAL is a graph-based representation which
+provides several advantages over matrix/block-based formats such as MAF, such as
+improved scalability and the ability to perform queries with respect to an
+arbitrary reference or subtree.
+
+This package includes the HAL API and several analysis and conversion tools
+which are described below. HAL files are presently stored in either HDF5 or mmap
+format, but we note that the tools and most of the API are format-independent,
+so other databases could be implemented in the future.
+"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+local_sonLibHash = '562a466e4570c93a647ffb10aa2628143b4ee9f4'
+sources = [
+    {
+        'source_urls': ['https://github.com/ComparativeGenomicsToolkit/hal/archive/refs/tags/'],
+        'filename': 'release-V%(version)s.tar.gz',
+        'extract_cmd': 'tar xvf %s && mv hal-release-V%(version)s hal'
+    },
+    {
+        'source_urls': ['https://github.com/ComparativeGenomicsToolkit/sonLib/archive/'],
+        'download_filename': '%s.tar.gz' % local_sonLibHash,
+        'filename': 'sonLib-%s.tar.gz' % local_sonLibHash,
```

Comment:
Maybe rename on download to `sonLib-<datestamp-that-corresponds-to-this-commit>-<short-commit>.tar.gz`, so it's easy to tell which downloaded source tarball is newer?